x-pack/plugins/dashboard_enhanced/kibana.json

```diff
@@ -3,12 +3,21 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["data", "uiActionsEnhanced", "embeddable", "dashboard", "share"],
-  "configPath": ["xpack", "dashboardEnhanced"],
+  "configPath": [
+    "dashboardEnhanced",
```

Comment:
breaks the path?

Reply:
great catch, I guess that happened on accident. Fixed